code/modules/food_and_drinks/recipes/processor_recipes.dm

```diff
@@ -0,0 +1,50 @@
+/datum/food_processor_process
+	var/input
+	var/output
+	var/time = 40
+	var/required_machine = /obj/machinery/processor
+
+/datum/food_processor_process/meat
+	input = /obj/item/reagent_containers/food/snacks/meat/slab
+	output = /obj/item/reagent_containers/food/snacks/faggot
+
+/datum/food_processor_process/bacon
+	input = /obj/item/reagent_containers/food/snacks/meat/rawcutlet
+	output = /obj/item/reagent_containers/food/snacks/meat/rawbacon
+
+/datum/food_processor_process/potatowedges
+	input = /obj/item/reagent_containers/food/snacks/grown/potato/wedges
+	output = /obj/item/reagent_containers/food/snacks/fries
+
+/datum/food_processor_process/sweetpotato
+	input = /obj/item/reagent_containers/food/snacks/grown/potato/sweet
+	output = /obj/item/reagent_containers/food/snacks/yakiimo
+
+/datum/food_processor_process/potato
+	input = /obj/item/reagent_containers/food/snacks/grown/potato
+	output = /obj/item/reagent_containers/food/snacks/tatortot
+
+/datum/food_processor_process/carrot
+	input = /obj/item/reagent_containers/food/snacks/grown/carrot
+	output = /obj/item/reagent_containers/food/snacks/carrotfries
+
+/datum/food_processor_process/soybeans
+	input = /obj/item/reagent_containers/food/snacks/grown/soybeans
+	output = /obj/item/reagent_containers/food/snacks/soydope
+
+/datum/food_processor_process/spaghetti
+	input = /obj/item/reagent_containers/food/snacks/doughslice
+	output = /obj/item/reagent_containers/food/snacks/spaghetti
+
+/datum/food_processor_process/corn
+	input = /obj/item/reagent_containers/food/snacks/grown/corn
+	output = /obj/item/reagent_containers/food/snacks/tortilla
+
+/datum/food_processor_process/parsnip
+	input = /obj/item/reagent_containers/food/snacks/grown/parsnip
+	output = /obj/item/reagent_containers/food/snacks/roastparsnip
+
+/datum/food_processor_process/mob/slime
+	input = /mob/living/simple_animal/slime
+	output = null
+	required_machine = /obj/machinery/processor/slime
```

Comment:
You may want to also make the food products unable to be created in the slime processor by making a second child

Reply:
That would introduce a dummy parent type that I'm not really keen on, and i feel we should keep the part where the slime processor is just science's hacky upgrade to the food processor to process slimes.

I don't think it matters either way, I don't think anyone uses a slime processor to process food.

Reply:
Dummy parents are perfectly fine code-wise, if anything it's better practice though we don't do it much in ss13 code.